### ISSUE TITLE
Broadcast: GraphQLLocatedError 'DurationFloat' object has no attribute 'startswith'

### ISSUE THREAD

**Issue by ColemanTom** [cylc/cylc-flow#6334]
### Description

When broadcasting to set the `execution time limit` I'm seeing the error `ERROR: [{'error': {'message': "'DurationFloat' object has no attribute 'startswith'", 'traceback': ["graphql.error.located_error.GraphQLLocatedError: 'DurationFloat' object has no attribute 'startswith'\n"]}}]`. This may be related to #6333 and #6222 
This happens if you have two or more namespaces and are using a ISO duration item.

### Reproducible Example

```
$ cylc broadcast -s 'execution time limit=PT1M' -n root -n root access_g4_rewind
ERROR: [{'error': {'message': "'DurationFloat' object has no attribute 'startswith'", 'traceback': ["graphql.error.located_error.GraphQLLocatedError: 'DurationFloat' object has no attribute 'startswith'\n"]}}]
```

The namespaces can be the same, it doesn't matter. If you have one namespace, there are no issues, 2+ it fails.

### Expected Behaviour

No failure

**Comment by oliver-sanders** [comment]
@ColemanTom, the issue here appears to be the duplicate `-n root` option. When I remove this, the command succeeds:

```console
$ cylc broadcast -s 'execution time limit=PT1M' -n root -n root generic         
ERROR: [{'error': {'message': "'DurationFloat' object has no attribute 'startswith'", 'traceback': ["graphql.error.located_error.GraphQLLocatedError: 'DurationFloat' object has no attribute 'startswith'\n"]}}]
$ cylc broadcast -s 'execution time limit=PT1M' -n root generic
Broadcast set:
+ [*/root] execution time limit=PT1M
```

**Comment by ColemanTom** [comment]
The duplicate is for demonstration. You can do any combination of things, just 2+ namespaces setting at least execution time limit, causes this failure.

**Comment by oliver-sanders** [comment]
Should be fixed by #6335. If you are able to test this it would help us get it in sooner.

**Comment by ColemanTom** [comment]
As I said, the `root` duplication was me demonstrating without knowing any namespaces. Here is a simple example.

```
$ cat flow.cylc
[scheduling]
    initial cycle point = 21200101T0000Z
    final cycle point = 21200101T0000Z
    [[graph]]
        PT6H = """
                @wall_clock => start & end
        """
[runtime]
    [[start,end]]
        script = true

$ cylc broadcast -s 'execution time limit=PT9M' -n start -n end basic
ERROR: [{'error': {'message': "'DurationFloat' object has no attribute 'startswith'", 'traceback': ["graphql.error.located_error.GraphQLLocatedError: 'DurationFloat' object has no attribute 'startswith'\n"]}}]
```

It is nothing to do with duplicate namespaces. Cylc (or something) must be resolving the PT9M to be a DurationFloat, which breaks GraphQL.

**Comment by ColemanTom** [comment]
I have run with your change, and same error by the way.

**Comment by oliver-sanders** [comment]
> I have run with your change, and same error by the way.

I tried with your example above. Could you post some details of how you reproduced this on the PR so I can investigate, thanks.

Note, you will need to restart your workflows with the patched version because it's a server-side fix not a client-side one.

**Comment by ColemanTom** [comment]
Ah, I missed updating `broadcast.py` yesterday. I had only updated `broadcast_mgr.py`. With your new `broadcast.py` in there too, the error has disappeared, whether the namespaces mentioned are duplicate or not. 

```
$ CYLC_VERSION=8.3.3 cylc broadcast -s 'execution time limit=PT9M' -n start -n end basic
Broadcast set:
+ [*/end] execution time limit=PT9M
+ [*/start] execution time limit=PT9M
$ CYLC_VERSION=8.3.3 cylc broadcast -s 'execution time limit=PT9M' -n start -n start basic
Broadcast set:
+ [*/start] execution time limit=PT9M
```

So, this looks like it can be closed when https://github.com/cylc/cylc-flow/pull/6335 is merged in.

**Comment by oliver-sanders** [comment]
Thanks for confirming.

**Comment by oliver-sanders** [comment]
Closed by #6335 (Cylc 8.3.4)
